k8s/spark-operator/README.md

```diff
@@ -152,24 +152,24 @@ expanded manifest file for future updates to the application.
 
 ```shell
 awk 'FNR==1 {print "---"}{print}' manifest/* \
-  | envsubst '$name $namespace $sparkOperatorImage $serviceAccount' \
-  > "${name}_manifest.yaml"
+  | envsubst '$NAME $NAMESPACE $SPARK_OPERATOR_IMAGE $SERVICE_ACCOUNT' \
```

Comment:
Should it be $NAME -> $APP_INSTANCE_NAME?

Reply:
Should be `APP_INSTANCE_NAME`.